runtime/layers/fpm/php-fpm.conf

```diff
@@ -17,7 +17,7 @@ clear_env = no
 catch_workers_output = yes
 ; New PHP 7.3 option that disables a verbose log prefix
 ; Disabled for now until we switch to PHP 7.3
```

Comment:
Could you remove that comment? (or update it to signal that this line will be removed on PHP 7.2?)

Reply:
sure